documentation/upgrading.md

```diff
@@ -1,6 +1,6 @@
 # Upgrade Guide
 
-This document provides guidelines for upgrading between versions of Flow PHP. 
+This document provides guidelines for upgrading between versions of Flow PHP.
```

Comment:
I would leave that file main folder and link in it to that new docs page, so if anyone looks at repo first, he will not be lost.

Reply:
I'm actually slowly trying to move away everything to website and turn it into main documentation source, thats why I'm fixing links.

Ideally I would like to keep just very minimalistic README.md on the root and put links from it to all important places like UPGRADING or CONTRIBUTING

My goal is to make the docs more interactive, with some cross referencing and mentions.